### ISSUE TITLE
Boegelbot should ask PR author to upload a test report

### ISSUE THREAD

**Issue by SebastianAchilles** [boegel/boegelbot#12]
If a PR has not test report after a given time (e.g. 24h) boegelbot should kindly ask the PR author to upload a test report. In the message a link to the documentation of `--upload-test-report` could be added: https://docs.easybuild.io/en/latest/Integration_with_GitHub.html#updating-existing-pull-requests-update-pr

**Comment by boegel** [comment]
Makes sense, but may not be trivial to implement. The bot currently doesn't keep any state, so how would it know which PRs it should check for missing test report? Consider all PRs that were updated in the last 24h (that's doable I think)?
